### ISSUE TITLE
Import Source fails when Fault Tolerance is switched on

### ISSUE THREAD

**Issue by idandr** [Icinga/icingaweb2-module-vsphere#18]
Since I have switched Fault Tolerance to ON for a virtual machine, "Import Source" fails with the message "Duplicate entry: <name_of_vm>". I can not import or update any machines at the moment (as long as I do not switch off Fault Tolerance for all virtual machines).

Info about Fault Tolerance mode: There is a primary (running) machine on one VMhost and a secondary (switched off) copy of this machine on another VMhost. Primary is syncing all changes to secondary.

Having a look at Import Source - Preview tab in director, this VM is listed twice with the same name. For one of the machines guest.guestState is set to "running", for the other one to "notRunning". The value of config.uuid is the same for both machines.

If there is no other information available to distinguish between primary and secondary virtual machine, I would suggest to import the running machine only for machines with matching name and config.uuid.

**Comment by MrSqu1nty** [comment]
Have you tried using a filter on the Sync?

You could try guest.guestState=notRunning although YMMV because a similar filter doesn't work for us.

**Comment by idandr** [comment]
Thanks for your suggestion. Unfortunately, I am stuck in the first step, "Import source". Sync would be the second.

But I found a solution: In the answer to #8 is a hint to a black/whitelist property modifier in Director. This new modifier (see https://github.com/Icinga/icingaweb2-module-director/issues/1371) is flaged for the not yet published release 1.5.0. I used the latest release (1.4.3) up to now. After an upgrade to git master, there is an import source modifier "Black or White-list rows based on property value". Using this modifier for field guest.guestState works well (after setting "Target property" to a variable name without nesting = without a dot in the name).
